pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -43,6 +43,8 @@ const (
 )
 
 func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *apps.Deployment {
+	logger.Debugf("mgrConfig: %+v", mgrConfig)
+	var tolerationSeconds int64 = 5
```

Comment:
5 seconds seems too aggressive. A network glitch could cause the pod to move around more often that someone might expect. The mgr pod isn't that critical. What about a toleration of 30 seconds?

Reply:
The node controller has a default timeout of  40s to start reporting ConditionUnknown. So the "migration" of the pod only will be possible after this 40 seconds + 5 seconds (in the Toleration). 
I think that a network glitch of this dimension (45 seconds)  is enough severe to be good for "migrate" all the pods in the node.

Reply:
Ok, makes sense. Could you add a comment here about that? thanks